tests/device/cpu/test_rapl.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2023 Jae-Won Chung <jwnchung@umich.edu>
```

Comment:
I guess we're better off removing the entire Apache-2.0 notice on source files 😅 Zeus used to be just me but now there are more than 10 legit contributors.